docs/04-operation-guides/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Did not receive an event - basic diagnostics
+---
+
+## Symptom
+
+You publish an event but the event is not received by the subscriber.
+
+## Remedy
+
+Follow these steps to detect the source of the problem:
+
+1. Check the eventing backend CRD, if eventingReady is true. Run the command:
```

Comment:
nit: IMHO this reads better but might also be personal choice (in that case please feel free to just reject it)
```suggestion
1. Check the eventing backend CRD if the field `eventingReady` is true:
```